components/DailyCrusher/DailyCrusher.js

```diff
@@ -0,0 +1,171 @@
+
+  if (!currentWorkout) {
+    return <p>No workout scheduled for today. Rest day!</p>;
+  }
+
+  const muscleGroups = [];
+
+  const enrichedExercises = currentWorkout.exercises.map((exercise) => {
+    const foundExercise = findExerciseById(exercises, exercise.exerciseId);
+    if (foundExercise) {
+      muscleGroups.push(...foundExercise.muscleGroups);
+      return { ...exercise, name: foundExercise.name };
+    }
+    return exercise;
+  });
+
+  const uniqueMuscleGroups = [...new Set(muscleGroups)];
+
+  const dailyCrusher = {
+    ...currentWorkout,
+    exercises: enrichedExercises,
+    muscleGroups: uniqueMuscleGroups,
+  };
+
+  return (
+    <StyledCrusherWrapper>
+      <StyledHeadline>Crusher of the Day 💪</StyledHeadline>
+      <StyledArticleWrap>
+        <StyledDescriptionText>
+          Get in shape with our crusher workouts! Every day we suggest a
+          different workout for you to try out.
+        </StyledDescriptionText>
+
+        <StyledLink href="/workoutlist">
+          <Workout workout={dailyCrusher} spotlightMode={true} />
+        </StyledLink>
+        <StyledPostionRelative>
+          {isCrusherDetailsVisible ? null : (
+            <StyledAbsoluteToggleButton onClick={toggleVisibility}>
+              Show More
+            </StyledAbsoluteToggleButton>
+          )}
+          <StyledPreviewFilter
+            $showMore={
+              isCrusherDetailsVisible ? "filter: none; color: black;" : null
+            }
+          >
+            <h3>{dailyCrusher.title}</h3>
+            <StyledDescriptionText>
+              {dailyCrusher.description1}
+            </StyledDescriptionText>
+          </StyledPreviewFilter>
+        </StyledPostionRelative>
+        {isCrusherDetailsVisible ? (
+          <>
+            {dailyCrusher.exercises.map((exercise) => {
+              return (
+                <StyledExerciseSetsReps key={uid()}>
+                  <StyledBoldText>{exercise.name}: </StyledBoldText> Sets:{" "}
+                  {exercise.sets} / Reps: {exercise.reps}
+                </StyledExerciseSetsReps>
+              );
+            })}
+            <StyledDescriptionText>
+              {dailyCrusher.description2}
+            </StyledDescriptionText>
+            <StyledDescriptionText>
+              {dailyCrusher.description3}
+            </StyledDescriptionText>
+          </>
+        ) : null}
+      </StyledArticleWrap>
+      {!isCrusherDetailsVisible ? null : (
+        <StyledToggleButton onClick={toggleVisibility}>
+          Show Less
+        </StyledToggleButton>
+      )}
+    </StyledCrusherWrapper>
+  );
+}
+
+const StyledCrusherWrapper = styled.article`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  border-radius: 12px;
+  max-width: 600px;
+  margin: 20px auto;
+  text-align: center;
+  box-shadow:
+    0 4px 8px #0000001a,
+    0 -0.5px 5px #0000000d;
+`;
+
+const StyledArticleWrap = styled.div`
+  padding: 0 2rem;
```

Comment:
I think rem is great for font sizes etc., but illogical for spacing such as padding. If I imagine that I have a visual impairment and my default font size is very large, then I want to be able to read the font well, but I don't want to have much larger spacing – otherwise I might not be able to read much on a small screen. The design might be a little worse, but accessibility will be better, and that's what's important when using rem etc.

Reply:
We prefer rem as our unit for most of the cases. We also tested out what would happen if we increase the global font size to see what happens in terms of the padding and are fine with how it looks. Good point though, will keep that in mind :)